sos/plugins/xfs.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
             for e in dev:
                 parts = e.split(' ')
                 self.add_cmd_output("xfs_info %s" % (parts[1]))
+                self.add_cmd_output("xfs_admin -l -u %s" % (parts[1]))
```

Comment:
What's the value of adding a special XFS command to gather information that is already captured by the generic `blkid` and `lsblk` data?

```
# xfs_admin -l -u /dev/mapper/vg_hex-lv_images 
[sudo] password for breeves: 
label = "IMAGES"
UUID = 63f758a7-3ced-4cbc-9be9-3b766dafeaf8
```
vs.

```
# blkid /dev/mapper/vg_hex-lv_images
/dev/mapper/vg_hex-lv_images: LABEL="IMAGES" UUID="63f758a7-3ced-4cbc-9be9-3b766dafeaf8" TYPE="xfs"
```